generator/generators/Oracle/generator.ts

```diff
@@ -0,0 +1,122 @@
+import * as fs from "fs";
+import { createSourceFile, ScriptTarget, SyntaxKind } from "typescript";
+import { getAST } from "../../parsers/oracle/parser";
+import { transform } from "../../transformers/do/transformer";
+import { getDir,printFile } from "../lib/helper";
+
+interface FunctionData {
+  functionName: string;
+  SDKFunctionName: string;
+  params: param[];
+}
+
+interface param {
+  name: string;
+  type: string;
+  typeName: string;
+}
+
+interface ClassData {
+  className: string;
+  functions: FunctionData[];
+  serviceName: string;
+}
+
+const dummyFile = process.cwd() + "/dummyClasses/do.js";
+
```

Comment:
Why you are using the dummy class of digital ocean here.Is the dummy class for Oracle same as the class of digital ocean?